codalab/apps/web/static/js/worksheet/worksheet_side_panel.js

```diff
@@ -34,9 +34,11 @@ var WorksheetSidePanel = React.createClass({
 
     getFocus: function() {
         // Return the state to show on the side panel
-        if (this.props.focusIndex == -1)
+        var focussedBunble = this.props.ws.state.items[this.props.focusIndex]
```

Comment:
oh. yeah my bad. Sorry my British influenced English does this now and then. I did not know that it is spelled as focused in US. Thanks for letting me know. :+1: